README.md

```diff
@@ -181,6 +181,7 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 
 #### RSS
 
+* [Fluent Reader](https://github.com/yang991178/fluent-reader) - A modern desktop RSS reader. [![Open-Source Software][OSS Icon] ![Freeware][Freeware Icon]](https://github.com/ViennaRSS/vienna-rss)
```

Comment:
@freddyli7 The open source URL is wrong.

Reply:
really? it is `https://github.com/yang991178/fluent-reader`, it works for me.

Reply:
@freddyli7 
![image](https://user-images.githubusercontent.com/1680273/89578027-22c6bb00-d864-11ea-8a95-d0605b781b65.png)